src/main.ts

```diff
@@ -68,6 +68,9 @@ async function main() {
     port: parseInt(process.env.PORT || '0', 10),
   });
 
+  process.on('uncaughtException', (err) => {
```

Comment:
This sets off my spidey-sense 🕸 

Why is this required?

Reply:
Been so long. I don't think it is. I've put it there as I was debugging something and it actually helped out. Must have forgotten to take it out.